run.sh

```diff
@@ -0,0 +1,57 @@
+# Argument Parsing
+echo "Positional Parameters"
+echo 'Github Project Repository: ' $1
+echo 'Github Project Repository Module Path: ' $2
+
+if [ "$1" == "" ]; then
+    echo "arg1 - github project repository"
+    exit
+elif [ "$2" == "" ]; then
+    echo "arg2 - github project module path"
+    exit
+fi
+
+# Setup Variables
+crnt=`pwd`
+working_dir=`dirname $0`
+project_repository=$1
+project_module_path=${crnt}/$2
+
+# TestRunner
+if [ ! -d "testrunner" ]; then
```

Comment:
Probably would be better if we only ran `mvn install` when we need to

Reply:
Is there a clean way to check if the plugin has already been installed? I couldn't find one online

Reply:
You should be able to check if Maven plugins are installed by seeing if it is in the local Maven repository, under ~/.m2/repository, following the standard Maven repository layout using its group ID, artifact ID, and version.

On the other hand, since testrunner should be quite small, is it so bad to just re-install it?